install/ui/src/freeipa/idviews.js

```diff
@@ -317,6 +317,10 @@ return {
                 $type: 'cert_textarea',
                 name: 'usercertificate'
             },
+            {
+                $type: 'sshkey',
+                name: 'ipasshpubkey'
```

Comment:
Should the field have ``label: '@i18n:objects.sshkeystore.keys'``, too?

Reply:
I think it is not necessary because sshkey widget has 
```
text: text.get('@i18n:objects.sshkeystore.show_set_key')
```